index.html

```diff
@@ -15,31 +15,294 @@
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+            </div>
+            <div class="cards">
+                <img src="/Images/Img Plaholder.png" alt="Store image">
+                <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                    <div class="Languages">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="aboutme-button">See Project</button>
+                    </div>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+
+        </section>
+        <section class="About-me">
```

Comment:
- [ ] I kindly suggest that you ensure that your about me paragraph has the same text alignment as the Figma.
- [ ] Please kindly change the color of your paragraph text to #42526E; to match the Figma
- [ ] Please kindly increase the font weight of the your About me header to match the Figma design
- [ ] Please kindly change the color of your About me header text to #172B4D; to match the Figma

| Your Project | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/219884907-570f3362-fb25-450e-aff5-8b01a4d21c21.png)  | ![image](https://user-images.githubusercontent.com/52934128/219884954-5d7194ae-cba1-4579-afe2-01054225e98e.png)  |

Reply:
Kindly ensure that your paragraph is centered with the button and the title as requested by. the previous reviewer, it was also requested to increase the `font-weight` of the title which haven't been addressed 
![Screenshot 2023-02-18 at 17 21 04](https://user-images.githubusercontent.com/45739963/219901844-56ec7bbe-313d-4e0c-82e5-78c2f5ff35e2.png)